ci/azure-pipelines.yml

```diff
@@ -54,8 +54,9 @@ jobs:
   displayName: macOS (Release)
   dependsOn: Release_Notes
   variables:
-    IMAGE_NAME: 'macos-12'
-    SUPBERBUILD_IMAGE_NAME: 'macos-12'
+    IMAGE_NAME: 'macos-13'
+    MACOSX_DEPLOYMENT_TARGET: 11.7
+    SUPBERBUILD_IMAGE_NAME: 'macos-13'
```

Comment:
:facepalm:

Reply:
It's astonishing, I looked many times into the code and overlooked that typo as well.